corehq/tests/noseplugins/cmdline_params.py

```diff
@@ -12,33 +23,11 @@ class CmdLineParametersPlugin(Plugin):
     parameters = {}
 
     def options(self, parser, env):
-        reuse_msg = " To be used in conjunction with REUSE_DB=1."
-        parser.add_option(
-            '--reset-db',
-            action="store_true",
-            default=False,
-            help=("Drop existing test dbs, then create and migrate new ones, but do not teardown "
-                  "after running tests. This is convenient when the existing databases are "
-                  "outdated and need to be rebuilt." + reuse_msg),
-        )
-        parser.add_option(
-            '--flush-db',
-            action="store_true",
-            default=False,
-            help=("Flush all objects from the old test databases before running tests.  Much "
-                  "faster than `--reset-db`." + reuse_msg),
-        )
-        parser.add_option(
-            '--migrate-db',
-            action="store_true",
-            default=False,
-            help="Migrate the test databases before running tests." + reuse_msg,
-        )
         parser.add_option(
-            '--teardown-db',
-            action="store_true",
-            default=False,
-            help="Skip database setup; do normal teardown after running tests." + reuse_msg,
+            '--reuse_db',
```

Comment:
nit: maybe it's just me, but underscores in command-line options drive me crazy. I think the convention would be `--reuse-db` or `--reusedb`. My preference is the latter because it's a tiny bit shorter/easier to type, but up to you.